src/lucky/mime_type.cr

```diff
@@ -26,7 +26,7 @@ class Lucky::MimeType
   end
 
   def self.known_formats : Array(Symbol)
-    accept_header_formats.values.uniq
+    accept_header_formats.values.uniq!
```

Comment:
Ameba suggested this for performance, but I'm not too sold on it. I'll disable if it seems to cause any issues.